fe/src/project/Seller.Portal/areas/Clients/components/ClientForm/ClientForm.js

```diff
@@ -152,7 +167,21 @@ function ClientForm(props) {
                             </FormControl>
                         </div>
                         <div className="field client-form__field-row">
-                            <Button type="submit" variant="contained" color="primary" disabled={state.isLoading || disable}>{props.saveText}</Button>
+                            {!saved ? (
+                                <Button type="submit" variant="contained" color="primary" disabled={state.isLoading || disable}>{props.saveText}</Button>
+                            ) : (
+                                <Button
+                                    type="submit" 
+                                    variant="contained" 
+                                    color="primary"
+                                    onClick={(e) => {
+                                        e.preventDefault();
+                                        NavigationHelper.redirect(props.newsUrl)
```

Comment:
**suggestion:** why redirect to newsUrl on ClientForm? Probably copy paste error.